### PR TITLE
Bugfix/edgeandfirefoxremovalscript jumpcloud

### PR DESCRIPTION
**Pull request by ashish-zluri**
The following changes are for Jumpcloud
1. Modified the existing script to remove Edge Extension 
2. Added a new script to remove firefox extension.

### CODE REVIEW

File: scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1 [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            foreach ($file in $extensionFiles) {
+                if ($file.Name -like "*zluribrowseragent@zluri.com.xpi*") {
+                    Write-Output "Removing extension $($file.FullName) from profile $($profile.Name)"
+                    
+                    try {
+                        # Attempt to remove the file
+                        Remove-Item -Path $file.FullName -Force -ErrorAction Stop
+                        # Verify if the file still exists
+                        if (Test-Path $file.FullName) {
+                            Write-Output "Failed to remove extension $($file.FullName) - File still exists."
```
Comment (ashish-zluri) [inline scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1:36]: @sashank-zluri @Dhanush-akrity Callout here
This function will only work if the firefox browser is not open on the machine
While testing I saw we get "Access Denied" error for this statement if the firefox browser is open

**Review by dhanush-zluri — APPROVED** [review]

File: scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1 (hunk 1 above)
Comment (dhanush-zluri) [reply scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1:36]: @ashish-zluri Please check how we can force quit this.

File: scripts/edge/jumpcloud/windows/v1/zluri-windows-edge-remove-jumpcloud.ps1 [hunk 2]
```diff
@@ -1,41 +1,50 @@
...
+    } catch {
+        $noMore = 1
+    }
+} until ($noMore -eq 1)
 
-return "DONE"
\ No newline at end of file
+$extensionCheck = $extensionsList | Where-Object { $_.Value -eq $edgeRegistryEntry }
+
+if ($extensionCheck) {
```
Comment (sashank-zluri) [inline scripts/edge/jumpcloud/windows/v1/zluri-windows-edge-remove-jumpcloud.ps1]: Update this name

File: scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1 (hunk 1 above)
Comment (sashank-zluri) [reply scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1:36]: We shouldn't force quit ig. 

Quit with an option ?  

Force quit isnt a viable option IMO.

**Review by sashank-zluri — CHANGES_REQUESTED** [review]

File: scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1 (hunk 1 above)
Comment (dhanush-zluri) [reply scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1:36]: Force quit extension. Not browser

Comment (ashish-zluri) [reply scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1:36]: @dhanush-zluri How to force quit extension ?

Comment (ashish-zluri) [reply scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1:36]: Even though in our scripts we are not even adding the extension to this path the extension is getting added here
Force quit browser is one option I see, but that isn't a good user experience.

Should be mark this as callout itself in our docs that browser should be closed when doing this step ?

**Comment by sashank-zluri** [comment]
Open a ticket @ashish-zluri  for this.

**Pull request merged**